src/VulkanSample.cpp

```diff
@@ -261,16 +262,30 @@ struct CommandLineParameters
     }
 } g_CommandLineParameters;
 
-void SetDebugUtilsObjectName(VkObjectType type, uint64_t handle, const char* name)
+void SetDebugUtilsObjectName(VkObjectType type, uint64_t handle, std::string name)
 {
-    if(vkSetDebugUtilsObjectNameEXT_Func == nullptr)
+    TEST(!name.empty());
```

Comment:
Why do you want to disallow empty names? I wouldn't add such check.

Reply:
I agree. I was introducing the policy that every Vulkan object has a non-empty name in my own codebase and applied this here as well. But Vulkan spec does not say this. I will remove that.